op-batcher/compressor/config.go

```diff
@@ -16,6 +16,9 @@ type Config struct {
 	// Kind of compressor to use. Must be one of KindKeys. If unset, NewCompressor
 	// will default to RatioKind.
 	Kind string
+
+	CompressionAlgo string
+	CompressLevel   int
```

Comment:
Would be better to reuse a single type from `derive` like
```suggestion
	Algo CompressionAlgo
```